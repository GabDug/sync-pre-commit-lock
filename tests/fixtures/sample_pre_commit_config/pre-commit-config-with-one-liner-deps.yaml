
# Many unused lines before document separator

---
default_language_version:
  python: python3.11

repos:

  - repo: https://github.com/pre-commit/mirrors-mypy
    # Some comment
    rev: v1.0.0
    hooks:
      - id: mypy
        additional_dependencies: [types-PyYAML, types-requests]
