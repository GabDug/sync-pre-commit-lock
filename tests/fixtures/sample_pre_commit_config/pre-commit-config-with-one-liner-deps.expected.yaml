
# Many unused lines before document separator

---
default_language_version:
  python: python3.11

repos:

  - repo: https://github.com/pre-commit/mirrors-mypy
    # Some comment
    rev: v1.5.0
    hooks:
      - id: mypy
        additional_dependencies: [types-PyYAML==1.2.4, types-requests==3.4.5]
